.drone.yml

```diff
@@ -42,6 +42,13 @@ pipeline:
       - git submodule update --init --recursive --jobs 7
       - ./project/scripts/sbt community-build/test
 
+  test_bootstrapped_java11:
+    group: test
+    image: lampepfl/dotty:2019-04-22
+    commands:
+      - cp -R . /tmp/6/ && cd /tmp/6/
+      - JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64/" ./project/scripts/sbt -java-home "/usr/lib/jvm/java-11-openjdk-amd64/" ";dotty-bootstrapped/compile ;dotty-bootstrapped/test; dotty-semanticdb/compile; dotty-semanticdb/test:compile;sjsSandbox/run;sjsSandbox/test"
```

Comment:
I think instead of passing java home the simplest thing to do is set the PATH to start with "/usr/lib/jvm/java-11-openjdk-amd64/bin